src/lib/list.ts

```diff
@@ -9,5 +9,5 @@ export async function list(this: Connection): Promise<TableMetadata[]> {
     `${this.options.host}/chain/${this.options.chainId}/tables/controller/${address}`
   ).then((r) => r.json());
 
-  return camelCaseKeys(res);
```

Comment:
Not sure I understand _why_ this fixes the issues. Was camelCaseKeys causing the issue, was it simply not needed anymore, or both?

Reply:
Both haha,
the camelCaseKeys function uses `Object.fromEntries` and `Object.entries` to turn an Object into an iterable (Array) change the keys and then convert back to an Object.  If you pass in an Array, javascript treats the Array like an Object with keys of '0', '1', '2', etc... The function converts them to lowerCamelCase, which in that case is the same value.  This results in an Object that has keys that look like an Array, e.g. {'0': 'foo'}.
So when the tests where checking if result[0] === 'foo' they were passing.

Basically that helper should only get called with an Object. I don't know a way to imply that via Typescript aside from `any`